data/weapons.txt

```diff
@@ -846,6 +846,62 @@ effect "torpedo fire"
 	"lifetime" 15
 
 
+outfit "Heat Bomb"
+	plural "Heat Bombs"
+	category "Ammunition"
+	cost 4000
+	thumbnail "outfit/torpedo"
+	"mass" .6
+	"heat bomb capacity" -1
+	description "Heat Bombs are ammunition for the Heat Bomb Launcher. Each Heat Bomb Launcher can hold 20 of them. You cannot install Heat Bombs unless you have also installed a launcher to fire them from."
+
+outfit "Heat Bomb Storage Rack"
+	category "Ammunition"
+	cost 60000
+	thumbnail "outfit/torpedo storage"
+	"mass" 3
+	"outfit space" -9
+	"torpedo capacity" 10
+	ammo "Torpedo"
```

Comment:
torpedo capacity —> heat bomb capacity
Torpedo —> Heat Bomb
Don’t want to be storing the wrong ammo.